README.md

```diff
@@ -5,40 +5,56 @@ Alpine Linux package builder for ROS (Robot Operating System)
 ## Build builder container
 
 ```shell
-docker build -t alpineros/ros-abuild:3.11-noetic .
+docker pull ghcr.io/alpine-ros/ros-abuild:3.11-noetic
 ```
-
-or pull from GitHub Container Registry
+or build locally
 
 ```shell
-docker pull ghcr.io/alpine-ros/ros-abuild:3.11-noetic
-docker tag ghcr.io/alpine-ros/ros-abuild:3.11-noetic alpineros/ros-abuild:3.11-noetic
+docker build -t ghcr.io/alpine-ros/ros-abuild:3.11-noetic .
```

Comment:
Considering the image tag, shouldn't the build command be like this?
```
docker build -t ghcr.io/alpine-ros/ros-abuild:3.11-noetic \
    --build-arg ROS_DISTRO=noetic \
    --build-arg ALPINE_VERSION=3.11 \
    --build-arg ROS_PYTHON_VERSION=3 .
```

Otherwise the package will be built for `kinetic`.